containers/hydra/Dockerfile

```diff
@@ -8,7 +8,7 @@ FROM nixos/nix
 RUN mkdir -p /setup/
 COPY channel.sh user.sh nix_conf.sh postgres.sh hydra.sh \
      populate.sh packages.lst upload.sh /setup/
-COPY hydra_conf.sh postbuild.py extracopy.sh /setup/
+COPY hydra_conf.sh postbuild.py extracopy.sh messager.py slack_config /setup/
```

Comment:
slack_config is not supposed to go the image (it's not supposed to be hardcoded in there), but user should be able to add it.
Nor does the file exist in the git repo, so building the image with this Dockerfile will simply fail.